src/test/java/pl/touk/sputnik/processor/shellcheck/ShellcheckProcessorTest.java

```diff
@@ -0,0 +1,116 @@
+package pl.touk.sputnik.processor.shellcheck;
+
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pl.touk.sputnik.TestEnvironment;
+import pl.touk.sputnik.configuration.ConfigurationBuilder;
+import pl.touk.sputnik.exec.ExternalProcess;
+import pl.touk.sputnik.review.Review;
+import pl.touk.sputnik.review.ReviewFile;
+import pl.touk.sputnik.review.ReviewFormatterFactory;
+import pl.touk.sputnik.review.ReviewResult;
+import pl.touk.sputnik.review.Severity;
+import pl.touk.sputnik.review.Violation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ShellcheckProcessorTest extends TestEnvironment {
+
+    private static final String CONFIGURATION_WITH_SHELLCHECK_ENABLED = "shellcheck/sputnik/noConfigurationFile.properties";
+    private static final String REVIEW_FILE_WITH_ONE_VIOLATION = "src/test/resources/shellcheck/testFiles/oneViolation.sh";
+    private static final String REVIEW_FILE_WITH_EXCLUDED_VIOLATION = "src/test/resources/shellcheck/testFiles/excludedViolation.sh";
+    private static final String REVIEW_FILE_WITH_MULTIPLE_VIOLATIONS = "src/test/resources/shellcheck/testFiles/multipleViolations.sh";
+    private ShellcheckProcessor sut;
+
+    @BeforeEach
+    public void setUp() {
+        config = ConfigurationBuilder.initFromResource(CONFIGURATION_WITH_SHELLCHECK_ENABLED);
+        sut = new ShellcheckProcessor(config);
+    }
+
+    @Test
+    public void shouldReturnNoViolationsWhenThereIsNoFileToReview() {
```

Comment:
Why there is no Assumption here?

Reply:
In that case, the file to review does not exist thus the processor is not invoked